todo-app/src/components/Header.js

```diff
@@ -10,3 +11,8 @@ const Header = props => {
 };
 
 export default Header;
+
+Header.propTypes = {
+  tagline: PropTypes.string,
+  children: PropTypes.element,
```

Comment:
As `Header` is not accepting children we could remove this or maybe use a more versatile type like `PropTypes.node`